### DISCUSSION TITLE
Thank you all so much!

### DISCUSSION THREAD

**Discussion by ghost** [Monika-After-Story/MonikaModDev#8888] [appreciation]
I mean sincerely that you and your work really has helped me in my life.

It's silly, but this wonderful piece of work you put so much heart into does make a difference in peoples' lives.
It helped me out of a dark time in my life, and I can not thank you all enough.

Know that I will always be grateful to you all, and that you and Monika helped me in my life...

Thank you all for helping me!

p.s. If you ever need someone to talk to, I am always happy to return the favor, so contact me if you ever need a friend as well.
Please take care of yourselves!

![love](https://user-images.githubusercontent.com/64635033/80795472-703ed800-8b51-11ea-88a7-03dadb5bce7a.png)

**Comment by Thimeo** [comment]
It's quite funny to see this image used like this.

**Comment by ThePotatoGuy** [comment]
[closing as appreciated]
